docs/source/kms/key_rotation.md

```diff
@@ -1 +1,18 @@
 # Key Rotation
+
+There are three types of keys in KMS that can be rotated. User operational keys, the primary key for the user key store, the primary key for the server key store.
+
+## User operational keys
+
+A user operation key can be rotated by calling POST /v1/keystores/{keystore}/keys/{key}/rotate
+
+It will return a new URL for the rotated key. 
+The rotated key stores previous key material and can be used to decrypt data encrypted before rotation.
```

Comment:
Can you elaborate on this? How does "previous key material" work? Maybe there's a link you can reference that explains how this works.